components-core/src/main/java/com/adyen/checkout/components/core/App.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ararat on 12/3/2024.
+ */
+
+package com.adyen.checkout.components.core
+
+import com.adyen.checkout.core.exception.ModelSerializationException
+import com.adyen.checkout.core.internal.data.model.ModelObject
+import com.adyen.checkout.core.internal.data.model.getStringOrNull
+import kotlinx.parcelize.Parcelize
+import org.json.JSONException
+import org.json.JSONObject
+
+@Parcelize
+data class App(
```

Comment:
`apps` could be used in the future for other payment methods as well, so `UPIApp` is a bit too specific. Maybe something like `AppData` or something generic like that? So that is doesn't look like an Android `Application`, but is also not payment method specific

Reply:
It is not easy to find a generic name for it. What do you think if we keep it `App`? It is in `com.adyen.checkout.components.core.App`, so the import will indicate that it is not an Android class.

Reply:
Changed it to `AppData`